book-edited.tex

```diff
@@ -21247,7 +21247,7 @@ \section{Keyboard Navigation}\label{keyboard-navigation}}
 \texttt{Tab}s. Note that hitting \texttt{Enter} when focused on a text
 entry clears the text entry; in most browsers, it submits the containing
 form instead. That quirk is a workaround for our browser
-{not implementing} the
+not implementing the
```

Comment:
Unnecessary ref?

Reply:
It’s how internal links end up compiler

Reply:
Right. I guess if doesn't change the output it's fine.